### PR TITLE
Add note about requiring a custom attribute for email template.

### LINKED ISSUE #154: [WIP] Enterprise email template example should explain that `datacenter` is a custom attribute
The email template example in https://sensuapp.org/docs/1.0/enterprise/integrations/email.html#custom-email-templates-example uses a custom client attribute `datacenter`. 

We should add a note indicating that this custom attribute must be explicitly set on the client and is not automatically configured, e.g. it is not inherited from "datacenter" defined in dashboard configuration.

### PR DESCRIPTION
**Pull request by rgeniesse**
## Motivation and Context
Closes #154.

## How Has This Been Tested?
Tested locally with `yarn`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New doc/guide
- [ ] Fixing errata
- [X] Update (Add missing or refresh existing content)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [x] All tests have passed.

### CODE REVIEW

**Comment by rgeniesse** [comment]
Initial review request as this needs to be added to the other SE versions.
- Should the addition of the `s` be done in a different pull request?
- Not sure if calling `datacenter` an object is proper here. Calling it an `attribute` when it is within the `@event` variable seemed awkward. Open to changing or making it the proper term.

**Comment by apaskulin** [comment]
- I usually fix typos in a separate commit than content changes, but include them in the same pull request. Since we're doing squash merges anyway, what you have here is 👌. 
- It seems like we do use `attribute` elsewhere. I'd love to hear more about why it seems awkward.
- Missing link, but assuming that's depending on #114

**Comment by rgeniesse** [comment]
- Yea, I almost did two commits, but then didn't due to the squash merge. :-)
- Awkward is probably the wrong word here. Maybe confusing? I just don't know if the proper term for `datacenter` is attribute or something else.
- Good catch on the link! I did have it, but then Atom's markdown thing was deleting white space on file save, which caused a bunch of tables to break. I then un-did everything trying to figure out why my additions were causing the breaking. I'll get that added back as I think #114 is already mostly in master already.

File: content/sensu-enterprise/2.6/integrations/email.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ ERB templating syntax][5]._
 The following example demonstrates how to access the Sensu `@event` variable from
 a custom ERB template.
 
+_NOTE: This example includes the `datacenter` object which needs to be added
```
Comment (cwjohnston) [inline content/sensu-enterprise/2.6/integrations/email.md]: > for the `@event` variable to contain it

Ending the sentence with this reads strangely to me. Would this work?

> This example includes the `datacenter` attribute, which is only available to be used in a template when defined as a client custom attribute

File: content/sensu-enterprise/2.8/integrations/email.md [hunk 2]
```diff
@@ -295,3 +298,4 @@ example      | {{< highlight shell >}}"body": "/etc/sensu/email/body_template.er
 [3]:  #templates-attributes
 [4]:  /sensu-core/1.2/reference/events#event-data
 [5]:  https://docs.puppet.com/puppet/latest/lang_template_erb.html
+[6]:  /sensu-core/1.4/reference/clients/#custom-attributes
```
Comment (cwjohnston) [inline content/sensu-enterprise/2.8/integrations/email.md]: should be core 1.2

File: content/sensu-enterprise/2.7/integrations/email.md [hunk 3]
```diff
@@ -286,3 +289,4 @@ example      | {{< highlight shell >}}"body": "/etc/sensu/email/body_template.er
 [3]:  #templates-attributes
 [4]:  /sensu-core/1.0/reference/events#event-data
 [5]:  https://docs.puppet.com/puppet/latest/lang_template_erb.html
+[6]:  /sensu-core/1.4/reference/clients/#custom-attributes
```
Comment (cwjohnston) [inline content/sensu-enterprise/2.7/integrations/email.md]: should be core 1.1

File: content/sensu-enterprise/2.6/integrations/email.md [hunk 4]
```diff
@@ -286,3 +289,4 @@ example      | {{< highlight shell >}}"body": "/etc/sensu/email/body_template.er
 [3]:  #templates-attributes
 [4]:  /sensu-core/1.0/reference/events#event-data
 [5]:  https://docs.puppet.com/puppet/latest/lang_template_erb.html
+[6]:  /sensu-core/1.4/reference/clients/#custom-attributes
```
Comment (cwjohnston) [inline content/sensu-enterprise/2.6/integrations/email.md]: I think we need Enterprise docs to reference material for the version of Sensu Core that correlates to the Enterprise version. Here this should be Core 1.0

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**